cache.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+type Cache interface {
+	read()
```

Comment:
For an interface to be exposed the functions inside it should be exported. All these are private as of now. Please make them public/exported

Reply:
Yeah will do that.